### ISSUE TITLE
CSIP9: I don't agree with MoSCoW classification

### ISSUE THREAD

**Issue by hsilva-keep** [DILCISBoard/E-ARK-CSIP#122]
https://dilcisboard.github.io/E-ARK-CSIP/implementation/#41-folder-structure-of-the-cs-ip
`CSIP9: The Information Package folder MUST include a folder named representations`

Following are just two scenarios where this "requirement" does not apply:
- When updating an AIP, using an update SIP, and the update only contains a metadata file (representations folder will be empty just to comply)
- An AIP with just metadata (representations folder will be empty just to comply)

Bottom line, I think this "requirement" **MUST** should be a **CAN**.

**Comment by karinbredenberg** [comment]
Connects to issue #121 empty folders shows the structure. Closing without action.
